habana/tensorflow/buildspec-2-8.yml

```diff
@@ -1,8 +1,8 @@
 account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
 region: &REGION <set-$REGION-in-environment>
 framework: &FRAMEWORK tensorflow
-version: &VERSION 2.5.0
```

Comment:
Can we follow the new naming convention for the older buildspec files?

Reply:
I think it's typo in this PR, has been fixed in 1.4.1 PR